app/uk/gov/hmrc/pdfgenerator/service/PdfGeneratorService.scala

```diff
@@ -23,8 +27,12 @@ class PdfGeneratorService @Inject()(configuration: Configuration, resourceHelper
   val CONFIG_KEY = "pdfGeneratorService."
 
   // From application.conf or environment specific
+
+  val RUN_MODE = configuration.getString(CONFIG_KEY + "runmode").getOrElse("prod").toLowerCase
+  val VALID_GOVUK_REGEX: String = configuration.getString(CONFIG_KEY + "validGovRegex").getOrElse("https://[a-z.]*gov.uk[/a-z0-9-]*")
```

Comment:
Regular expression wouldn't work for:
- locahost
- urls with - before gov.uk
- urls with underscores
- urls with query parameters
- urls with capital letters

Reply:
We can improve the regex for these scenarios.